client/project_test.go

```diff
@@ -23,20 +24,28 @@ var _ = Describe("Project", func() {
 		BeforeEach(func() {
 			mockOrganizationIdCall(organizationId)
 
-			httpCall = mockHttpClient.EXPECT().
-				Post("/projects", map[string]interface{}{
-					"name":           projectName,
-					"organizationId": organizationId,
-					"description":    projectDescription,
+			payload := struct {
+				ProjectCreatePayload
+				OrganizationId string `json:"organizationId"`
+			}{
+				ProjectCreatePayload{
+					Name:            projectName,
+					Description:     projectDescription,
+					ParentProjectId: parentProjectId,
```

Comment:
why not adding a case for a parent project, and keep the one without? both cases are valid, we just need to make sure that the one with parent ALSO works

Reply:
I'm not sure there will be added value in this.
I added it because the test would have failed without my fix. But with the fix there's little value with way.